extractor/src/main/scala/com/digitalasset/extractor/writers/postgresql/Queries.scala

```diff
@@ -10,6 +10,9 @@ import com.daml.extractor.Types._
 import com.daml.extractor.ledger.types._
 import doobie._
 import doobie.implicits._
+// legacy versions required for postgres see https://github.com/tpolecat/doobie/releases/tag/v0.8.8
```

Comment:
This seems like a very confusing API. Postgres requires importing a legacy API? :thinking:

I guess you have to use oracle if you want the non-legacy API?

Reply:
🤷‍♂️ tpolecat/doobie#1236 tpolecat/doobie#1099